### PR TITLE
clear cache

### PR DESCRIPTION
**Pull request by borisnadion**

### CODE REVIEW

File: utils/system.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+import re
+
+# supposed to work on windows
+def parse_params():
```
Comment (delip) [inline utils/system.py]: This is a non-standard way of handling cmd line args. Mind using argparse? 
https://docs.python.org/2/library/argparse.html

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 tqdm
-gensim
 sklearn
 numpy
 scipy
 nltk
+argparse
```
Comment (delip) [inline requirements.txt]: argparse is part of standard python library. no need to add to requirements.

File: utils/system.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import sys
+import os
+import re
+import argparse
+
+# supposed to work on windows
+def parse_params():
+    parser = argparse.ArgumentParser(description='FakeNewsChallenge fnc-1-baseline')
+    parser.add_argument('-c', '--clean-cache', action='store_true', default=False, help="clean cache files")
+    parser.add_argument('-d', '--dry-run', action='store_true', default=False, help="don't actually clean cache files")
```
Comment (delip) [inline utils/system.py]: pls remove --dry-run. It's not like we're launching a missile 😉

**Review by delip — CHANGES_REQUESTED** [review]

File: utils/system.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                os.remove(fname)
+    for f in ['hold_out_ids.txt', 'training_ids.txt']:
+        fname = os.path.join('splits', f)
+        if os.path.isfile(fname):
+            if params.dry_run:
+                print("Will delete {0}".format(fname))
+            else:
+                os.remove(fname)
+    print("All clear")
+    sys.exit(0)
```
Comment (delip) [inline utils/system.py]: why sys.exit(0) if --clean-cache is provided?

Shouldn't it clean the cache and continue with training by rebuilding the feature files?

**Review by delip — CHANGES_REQUESTED** [review]

File: utils/system.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import sys
+import os
+import re
+import argparse
+
+# supposed to work on windows
```
Comment (delip) [inline utils/system.py]: pls remove this comment

**Review by delip — APPROVED** [review]
Looks good. Pls remove that comment before merging.

File: utils/system.py (hunk 5 above)
Comment (borisnadion) [reply utils/system.py]: done

**Review by j6mes — APPROVED** [review]
ok

**Pull request merged**